Resources/Prototypes/Entities/Structures/Power/Generation/teg.yml

```
@@ -176,6 +176,7 @@
           nodeGroupID: Teg
 
     - type: AtmosUnsafeUnanchor
+    #- type: PipeRestrictOverlap # Allow pipe stacking
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)